src/containers/GraphLayers/GraphLayers.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
```

Comment:
Perhaps some comments in here to describe what this "layers" concept is about?  Also, why are the other toggles not included here?

Reply:
The others were not included because they didn't exist when I started this Redux experiment. Now that we have a couple more the "layers" concept here is pretty clear.